climakitae/threshold_tools.py

```diff
@@ -410,17 +394,17 @@ def conf_int(
     return conf_int_lower_limit, conf_int_upper_limit
 
 
-#####################################################################
+# ==============================================================================
 
 
 def get_return_value(
     ams,
-    return_period=10,
-    distr="gev",
-    bootstrap_runs=100,
-    conf_int_lower_bound=2.5,
-    conf_int_upper_bound=97.5,
-    multiple_points=True,
+    return_period = 10,
+    distr = "gev",
+    bootstrap_runs = 100,
+    conf_int_lower_bound = 2.5,
+    conf_int_upper_bound = 97.5,
+    multiple_points = True,
 ):
```

Comment:
from conf_int and down, the functions that have many items in the definition call have their ending `):` in line with the def call, is this right?